src/modm/platform/flash/stm32/flash.cpp.in

```diff
@@ -154,6 +154,7 @@ Flash::program(uintptr_t addr, MaxWordType data)
 	FLASH->CR &= ~FLASH_CR_PG;
 	return FLASH->SR & FLASH_SR_PGERR;
 %% else
+	FLASH->SR |= FLASH_SR_EOP;
```

Comment:
Are you sure this is necessary?

![image](https://user-images.githubusercontent.com/2820734/158413462-b1e4f782-ae4d-478c-9448-bd854b39d922.png)

Reply:
For the stm32g0 p. 1390 it should be cleared by software. Similarly for g4 devices it should be cleared p. 2124

Reply:
My screenshot is actually from the STM32G4 reference manual (RM0440, see edit above), but that is not the first ambiguity in ST documents...